### DISCUSSION TITLE
Plasma protein scaling factor(PPSF)

### DISCUSSION THREAD

**Discussion by nasirkalam** [Open-Systems-Pharmacology/Forum#769] [Questions & Problems] [answered]
Dear All,
Kindly tell me how to use option of plasma protein scaling factor in PK-Sim(R) ? Concentrations of Albumin./AAG have reduced in diseased state which results in increase of Fu of molecule. By default value of plasma protein scaling factor in PK-Sim individuale is 1.00. What to do if con. of Albumin/AAG reduced? how to simulate this situation in PK Sim? 
![image](https://user-images.githubusercontent.com/47519761/87664745-37d98e00-c77f-11ea-8dd1-20bf687aad3b.png)

1) Change the default value of plasma protein scaling factor from 1.00 to fraction of Albumin/AAG in diseased state e.g. in liver cirrhosis it reduced upto the fraction  0.3
or
2) Calculate the value of Fu in diseased state by using equation explained by Edginton etal., 2008. The fractional increase in Fu of diseased population from healthy will be used as plasma protein scaling factor.
![image](https://user-images.githubusercontent.com/47519761/87664832-5c356a80-c77f-11ea-8b1b-c2b6ea23a27e.png)

or 
3) Changing the Altered Fu value in molecule and leaving the scaling factor 1.00 by default.

**Comment by msevestre** [comment, marked as answer]
@nasirkalam This parameter is nothing more than an easy way for you to adjust the fu based for example on disease state etc..
Look at the description of the parameter
![image](https://user-images.githubusercontent.com/1041237/87680062-f5965980-c74a-11ea-8119-683e1105fbcc.png)

In my example, I have the following fu

![image](https://user-images.githubusercontent.com/1041237/87680155-1068ce00-c74b-11ea-857f-aa2350a63e75.png)


If  I multiply the scale factor by 2, the fu will also be multiplied accordingly
![image](https://user-images.githubusercontent.com/1041237/87680233-26768e80-c74b-11ea-927c-6f1862ff5d87.png)

so I'd suggest using option 1.

**Comment by nasirkalam** [comment]
Thanks Micheal Sevestre,
@msevestre 
Kindly make it more easy for me now i have two problems 
1) In my case, its not as true as you described. There was no change occur in Fu by changing PPSF
PPSF=1.00 (Default)
![image](https://user-images.githubusercontent.com/47519761/87707334-60cb4480-c7ba-11ea-8936-dccc03ee7c9c.png)
PPSF=0.3 
![image](https://user-images.githubusercontent.com/47519761/87707940-5c535b80-c7bb-11ea-936d-6b01847f4122.png)

 2) From your answer its obvious that if we need greater value of Fu we have to increase the PPSF  
![image](https://user-images.githubusercontent.com/47519761/87707634-dc2cf600-c7ba-11ea-8b84-052f5276662c.png)
Therefore Method 2 was more appropriate.
looking for your support.
Regards,
